proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -42,6 +42,11 @@ message RepositoryCommit {
   repeated buf.alpha.registry.v1alpha1.RepositoryTag tags = 8;
   // The commit's draft name, if it is a draft commit.
   string draft_name = 9;
+  // license_spdx_id is the license of the commit, based on the
+  // license file pushed, which should be one of the identifier defined in
+  // https://spdx.org/licenses, and will be empty if the license file is not
```

Comment:
Nit: "not set" over "empty" because the empty string be sent over the wire and may confer meaning.